example/src/RoomPage.tsx

```diff
@@ -8,7 +8,7 @@ import { useNavigate, useLocation } from 'react-router-dom';
 
 export const RoomPage = () => {
   const [numParticipants, setNumParticipants] = useState(0);
-  const [displayOptions, setDisplayOptions] = useState<DisplayOptions>({
+  const [displayOptions] = useState<DisplayOptions>({
```

Comment:
you can just use regular variable without using state if it's just constant data that never changed